### PR TITLE
Turn on 2nd timeline

### PR DESCRIPTION
**Pull request by austinjalexander**
# PR NUMERO UNO!

haha

## 2 Channels

image

Let me know if you want me to remove any of the minor config/doc changes.

There are some known issues with gulp 4, so I didn't want to deal with it here, but we might want to plan to address:
image

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+8.11.2
```
Comment (austinjalexander) [inline .nvmrc]: Just used this one; let me know if it should be diff.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
-# http
+# marvin-web
+
+https://marvin.paces-apache.space/
 
 ### Local Dev
+Requires `.dev.env` file.
```
Comment (austinjalexander) [inline README.md]: Realized this after it yelled at me.

File: README.md [hunk 3]
```diff
@@ -1,13 +1,19 @@
...
+
 ```sh
+nvm use
+npm i
 gulp serve
 ```
 
 ### Dockerize
 ```sh
-./scripts/build-image.sh
```
Comment (austinjalexander) [inline README.md]: The file seems to have this name, but let me know if I'm missing something.

File: package.json [hunk 4]
```diff
@@ -6,6 +6,10 @@
     "url": "https://github.com/cameronwp/marvin-web"
   },
   "version": "0.0.8",
+  "engines": {
+    "node": "8.11.2",
```
Comment (austinjalexander) [inline package.json]: Again, let me know if you want diff versions.

File: webpack.config.js [hunk 5]
```diff
@@ -7,7 +7,7 @@ const webpack = require('webpack');
 if (process.env.NODE_ENV === 'production') {
   env('../heart-of-gold/.env');
 } else {
-  env('../heart-of-gold/.dev.env');
+  env('./.dev.env');
```
Comment (austinjalexander) [inline webpack.config.js]: So, I was thinking it made sense to have the env file inside the repo, but let me know if you want to keep it in the parent.

File: README.md [hunk 6]
```diff
@@ -1,13 +1,19 @@
-# http
+# marvin-web
+
+https://marvin.paces-apache.space/
```
Comment (austinjalexander) [inline README.md]: Just realized this repo is public; not sure if you want to leave this here (or make this repo private).

File: webpack.config.js (hunk 5 above)
Comment (cameronwp) [reply webpack.config.js]: I've found the parent simplified deploying the whole cluster of Marvin apps. Can we `or` the env files? Look for the .env locally, if not found, then look at the parent?

File: README.md (hunk 6 above)
Comment (cameronwp) [reply README.md]: no. let's keep out anything specific to a given project. let's talk about this repo later

File: README.md (hunk 3 above)
Comment (cameronwp) [reply README.md]: no I think you're right. ty

File: webpack.config.js (hunk 5 above)
Comment (austinjalexander) [reply webpack.config.js]: We could use `fs.existsSync()`, or let `env` blow up and then catch it, but I think it's fine just to revert this for now; we can come back to it later.  👍

Comment (austinjalexander) [reply webpack.config.js]: Oh, also, might be good for us to specify what's in the env file, haha. Mine is blank atm. 🤷‍♂️

**Pull request merged**